lib/fpm/package/python.rb

```diff
@@ -99,17 +103,28 @@ def input(package)
 
     if File.directory?(path_to_package)
       setup_py = File.join(path_to_package, "setup.py")
+      pyproject_toml = File.join(path_to_package, "pyproject.toml")
     else
       setup_py = path_to_package
+      pyproject_toml = path_to_package
     end
 
-    if !File.exist?(setup_py)
-      logger.error("Could not find 'setup.py'", :path => setup_py)
-      raise "Unable to find python package; tried #{setup_py}"
+    # @todo Swap it! - TOML priority is for debugging only!
```

Comment:
It is time to swap

Reply:
As setup.py project slowly dying, I decide to keep priority of new behavior over old one.